### ISSUE TITLE
DSL : Export changelog file add json at the end of the text.

### ISSUE THREAD

**Issue by flesire** [nemerosa/ontrack#486]
I retrieve file changes with the following DSL script ;
```groovy
println '### Database changes :'
changelog.files.each {
    if(it.path.contains(".sql")) {
      println "${it.path}"
    }
}
```
The script return the correct list of file but it print also the json object after these lines.

I use ontrack-dsl-shell with Ontrack 2.28.2.

Remark : Same problem with commit list and exportIssues method.

**Comment by dcoraboeuf** [comment]
Quick work-around: add `0` or `''` (empty string) at your script.
By default, Groovy will return the last object.

**Comment by dcoraboeuf** [comment]
It's actually a feature, the DSL shell is supposed to return the computed value, so that you can link it to other processing (`jq` for example).

**Comment by dcoraboeuf** [comment]
I've added a bit more documentation about the JSON output.

**Comment by dcoraboeuf** [comment]
Will be available in 2.29 at the end of January.

**Comment by flesire** [comment]
Thanks.
